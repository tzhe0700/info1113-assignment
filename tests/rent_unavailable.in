ADD STUDENT Harry Potter
ADD STUDENT Hermione Granger
ADD COLLECTION spellbooks.csv
RENT 100000 111111
RENT 100001 111111
EXIT

