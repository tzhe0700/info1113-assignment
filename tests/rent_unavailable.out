user: Success.

user: Success.

user: 1 spellbooks successfully added.

user: Success.

user: Spellbook is currently unavailable.

user: Ending Archive process.

